src/layers/normalise.jl

```diff
@@ -166,20 +166,20 @@ end
 
 function LayerNorm(sz, λ=identity; affine=true, ϵ=1f-5)
   sz = sz isa Integer ? (sz,) : sz
```

Comment:
Tidying up?
```suggestion
function LayerNorm(sz, λ=identity; affine::Bool=true, ϵ::Real=1f-5)
```
Also, do we really have a public keyword called `ϵ`? I thought they were supposed to be ascii...

I guess "`sz` should be an integer or a tuple of integers." is a bit ugly too, can't it just be integers?